storio-sample-app/src/main/java/com/pushtorefresh/storio/sample/ui/fragment/TweetsFragment.java

```diff
@@ -176,4 +181,63 @@ public void onCompleted() {
                     }
                 });
     }
+
+    /**
+     * This method from {@link com.pushtorefresh.storio.sample.ui.adapter.TweetsAdapter.OnUpdateTweetListener}
+     * interface.
+     * It updates specific tweet by adding '+' to the end of tweet author
+     * every time when is called.
+     * It has chain of 3 steps in ReactiveX-way:
+     * 1. getting tweet via its id
+     * 2. mapping with changing author
+     * 3. putting result back to database
+     */
+    @Override
+    public void onUpdateTweet(@NonNull final Long tweetId) {
+        // 1.
+        storIOSQLite
```

Comment:
We need to handle Subscription

Reply:
Actually we don't because we should put value even if activity has already stopped